packages/dynamite/dynamite/example/README.md

```diff
@@ -0,0 +1,26 @@
+# Setup
+
+The generated code makes use of various third party packages. Pleas add the following dependencies to your `pubspec.yaml`.
+```yaml
+dependencies:
+    - built_collection
+    - built_value
+    - collection
+    - dynamite_runtime
+    - meta
+    - universal_io
+    - uri
+dev_dependencies:
+    - build_runner
+    - built_value_generator
+    - dynamite
+```
+
+# Generating
+
+To generate code you'll have to invoke the `build_runner` with the following command:
+```sh
+dart run build_runner build
+``` 
+The builder will look for any files ending with either `.openapi.json` or `.openapi.yaml` and place the generated code next to the specifications in a file ending with  `.openapi.dart`.
```

Comment:
Should we also support `.openapi.yml`? I can already see issues about "dynamite not working" just because of the different extension.